tests/test_corona.py

```diff
@@ -278,19 +278,21 @@ def test_pad_cumulative_series_to_yesterday():
 
 def test_germany_get_population():
     germany = c.germany_get_population()
-    assert germany > 8200000
-    hamburg = c.germany_get_population("Hamburg")
+    assert germany.shape == (412, 38)
```

Comment:
I'm not sure about this assert for two reasons: 
1. the number of subregions depends on the definitions and vary. I saw 401 (Wikipedia) and 412 (here). I also saw 402 and 416, somewhere I don't remember now.
2. It's not clear where does 28 come from.

Reply:
Yeah that was a bit lazy of me. I changed it so that the tests are more specific now, it checks what matters, in this case the name of the index, that the population column exists, and that the subregions in the population data match up with those in the cases data.